packages/core/package.json

```diff
@@ -42,7 +42,7 @@
     "prepare": "bob build"
   },
   "peerDependencies": {
-    "react": "^16.13.1",
+    "react": ">=16.13.1 <17.0.3",
```

Comment:
If we start doing `>=16.13.1`, why should we stop at `17.0.3` ? Is there anything specific about this version or the next one that we know is incompatible with our SDK?

Reply:
It was a suggestion from @0xnm , I guess he's afraid of breaking changes in versions `> 17.0.3`. If you ask me I would rather go up to `18.0.0`

Reply:
Frankly I'm still not convinced that we need to do this move. Can't we instead use `^17.0.0` (it was released 9 months ago) and keep a "compatible with" syntax instead of playing with numbers without being able to ensure compatibility?

Reply:
we have 2 options here:

1. Remove upper bound. Then if something doesn't work, we will hear it from the customer.
2. Use upper bound - need then constantly maintain it and also there is a question: what upper bound we should use. Probably it is safe to blindly include patch releases, then maybe something `< 17.1` (to include any version of `17.0.x`), if this syntax is not supported, maybe we can use something like `< 17.0.999`?

Reply:
`17.1.0` will work but maybe there's no need to have the upper bound ...what do you both say ?

Reply:
+1 for dropping upper bound. We use very little of React Native APIs, unlikely they are going to be changed. The only risky part is native modules interface and interactions tracking, but in former case breaking change will probably be well known, in the latter we have to check it with every version anyway, since we rely on the private property.

Reply:
Then is settled...I know that @xgouchet shares this opinion already.